### ISSUE TITLE
Some 'payload' commands also req topic

### ISSUE THREAD

**Issue by Paul-Reed** [Steve-Mcl/node-red-contrib-cron-plus#34]
Some of the payload commands also necessitate a topic to be specified, unless they don't work.

For example;

`{"command":"remove-all-dynamic"}` needs topic `delete-all-dynamic`
`{"command":"status-all"}` needs topic `status-all`
```
[{"id":"d52bdeb9.ecc12","type":"cronplus","z":"dbe9689e.fc98f8","name":"","outputField":"payload","timeZone":"","persistDynamic":false,"commandResponseMsgOutput":"output1","outputs":1,"options":[],"x":530,"y":932,"wires":[["1ea827af.902428"]]},{"id":"1ea827af.902428","type":"debug","z":"dbe9689e.fc98f8","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"true","targetType":"full","statusVal":"","statusType":"auto","x":685,"y":930,"wires":[]},{"id":"afc4bee8.8cdf8","type":"inject","z":"dbe9689e.fc98f8","name":"add every10Min","props":[{"p":"payload","v":"{\"command\":\"add\",\"name\":\"every10\",\"topic\":\"every10\",\"type\":\"default\",\"expressionType\":\"cron\",\"expression\":\"0 0/10 * * * *\"}","vt":"json"},{"p":"topic","v":"","vt":"string"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"{\"command\":\"add\",\"name\":\"every10\",\"topic\":\"every10\",\"type\":\"default\",\"expressionType\":\"cron\",\"expression\":\"0 0/10 * * * *\"}","payloadType":"json","x":325,"y":910,"wires":[["d52bdeb9.ecc12"]]},{"id":"1d1070b4.ed186f","type":"inject","z":"dbe9689e.fc98f8","name":"remove-all-dynamic","props":[{"p":"payload"},{"p":"topic","vt":"str"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"delete-all-dynamic","payload":"{\"command\":\"remove-all-dynamic\"}","payloadType":"str","x":330,"y":960,"wires":[["d52bdeb9.ecc12"]]},{"id":"b559ba7d.205128","type":"inject","z":"dbe9689e.fc98f8","name":"no topic","props":[{"p":"payload"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","payload":"{\"command\":\"remove-all-dynamic\"}","payloadType":"str","x":290,"y":1000,"wires":[["d52bdeb9.ecc12"]]},{"id":"3d8c5d35.c510e2","type":"inject","z":"dbe9689e.fc98f8","name":"random topic","props":[{"p":"payload"},{"p":"topic","vt":"str"}],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"randomtopic","payload":"{\"command\":\"remove-all-dynamic\"}","payloadType":"str","x":310,"y":1040,"wires":[["d52bdeb9.ecc12"]]}]
```

**Comment by Paul-Reed** [comment]
My fault! I had set the command payload as a string instead of json object.

The node's example flow shown below has a couple of typos, so I'll leave this here to remind you.
The typos are that the `remove-all-dynamic` & `status-all` inject nodes need to be set to json object instead of string, and the topic entries deleted.

![error](https://user-images.githubusercontent.com/973580/111915053-5f020100-8a6c-11eb-87e8-1336c9f5dd7c.JPG)

**Comment by Steve-Mcl** [comment]
fixed in 46759da
will be in next release.
Thanks for the helpful feedback
